Documentation/MRTK_Configuration_Dialog.md

```diff
@@ -0,0 +1,125 @@
+# MRTK project configuration dialog
+
+The MRTK configuration dialog is displayed when Unity loads a project and it is determined that one or more configuration options needs the attention of the developer.
+
+![Apply Later Ignore](Images/ConfigurationDialog/ConfigurationDialogHeader.png)
+
+To apply the changes, click the **Apply** button. The **Later** button will defer the changes until the project is reloaded at a future time.
+
+> [!NOTE]
+> The configuration dialog will reappear if one or more of the reccomended settings is left unchecked. To prevent this from occurring, apply the desired options, then relaunch the dialog via  **Mixed Reality Toolkit** > **Utilities** > **Configure Unity Project** and click **Ignore**. This will prevent the configuration dialog from reappearing automatically.
```

Comment:
```suggestion
> The configuration dialog will reappear if one or more of the recommended settings is left unchecked. To prevent this from occurring, apply the desired options, then relaunch the dialog via  **Mixed Reality Toolkit** > **Utilities** > **Configure Unity Project** and click **Ignore**. This will prevent the configuration dialog from reappearing automatically.
```